doc/source/howtos/garbd_howto.rst

```diff
@@ -89,16 +98,11 @@ up. This example is using cluster information from the :ref:`ubuntu_howto`.
 
   # Optional Galera internal options string (e.g. SSL settings)
   # see http://galeracluster.com/documentation-webpages/galeraparameters.html
-  # GALERA_OPTIONS=""
+  # GALERA_OPTIONS="socket.ssl_cert=/etc/ssl/mysql/server-key.pem;socket.ssl_key=/$"
```

Comment:
ssl_key is empty, should it use /etc/ssl/mysql/server-key.pem

Reply:
Done.